lib/ain-evm/src/evm.rs

```diff
@@ -326,4 +350,33 @@ impl EVMServices {
 
         Ok(())
     }
+
+    pub fn counter_contract() -> Result<CounterContractInfo, Box<dyn Error>> {
```

Comment:
Needs comment to explicit that it returns the contract with the count already incremented

Reply:
Fixed: 49ec7c6bd99f38f299817819353c1a652bdc55a4